network/p2p/cache/node_blocklist_wrapper.go

```diff
@@ -0,0 +1,191 @@
+package cache
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/libp2p/go-libp2p/core/peer"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/id"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/storage/badger/operation"
+)
+
+// IdentifierSet represents a set of node IDs (operator-defined) whose communication should be blocked.
+type IdentifierSet map[flow.Identifier]struct{}
+
+// Contains returns true iff id ∈ s
+func (s IdentifierSet) Contains(id flow.Identifier) bool {
+	_, found := s[id]
+	return found
+}
+
+// NodeBlocklistWrapper is a wrapper for an `id.IdentityProvider` instance, where the
```

Comment:
What about calling it `NodeBlacklist` or `NodeBlockedList`? I found Blocklist is a bit confusing

Reply:
sorry. For me the term "blocklist" is also unfamiliar. Please see [my comment here](https://github.com/onflow/flow-go/pull/3344#discussion_r990353927) regarding the naming. Hope Peter will help us understand the terminology.